### PR TITLE
K8s cql proxy

### PR DESCRIPTION
**Pull request by david-joy-ds**
Added documentation on how to use CQL-Proxy with Kubernetes. 

- New folder k8s with config.yaml and cql-proxy.yaml for k8s
- Readme for k8s cql-proxy configuration

### CODE REVIEW

File: k8s/readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# CQL PROXY with Kubernetes
+
+Few things to keep a note of when creating the deployment file in kubernetes for running cql-proxy. 
+
+### 1. **Create cql-proxy.yaml**
+
+- **Args**:  adding the args in the right format. Make sure to add the Client ID and Client Secret for your Astra Database.   
+
+      command: ["./cql-proxy"]
+        args: ["--bundle=/tmp/scb.zip","--username=Client ID","--password=Client Secret"]
```
[the remark this answers is not in the document]
Comment (david-joy-ds) [reply k8s/readme.md:10]: sounds good.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,14 @@ or using Docker as
 docker run -p 9042:9042 --rm datastax/cql-proxy:v0.0.1 --contact-points <cluster node IPs or DNS names>
 ```
 
+or using Kubernetes as below. Check the k8s folder for `config` and `cql-proxy.yaml` and more documentation.
+
+```sh
+kubectl create -f cql-proxy.yaml
+```
```
[the remark this answers is not in the document]
Comment (david-joy-ds) [reply README.md:45]: changes made.

File: k8s/readme.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# CQL PROXY with Kubernetes
```
[the remark this answers is not in the document]
Comment (david-joy-ds) [reply k8s/readme.md:1]: added notes in readme for Astra DB.

File: k8s/readme.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            name: config        
+    
+    **Note** : Keep the volumne name and volumeMount name same.
+
+### 2. **Create a configmap**
+
+Use the secure bundle zip name as `scb.zip`. Get the secure bundle from the connect page in Astra Database. 
+      
+
+      kubectl create configmap config --from-file scb.zip
```
[the remark this answers is not in the document]
Comment (david-joy-ds) [reply k8s/readme.md:34]: Changes made.

**Comment by david-joy-ds** [comment]
Please review, changes have been made.

File: k8s/readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (phact) [reply k8s/readme.md:10]: @david-joy-ds looks like these can be env vars now

**Review by phact — APPROVED** [review]

**Review by dougwettlaufer — APPROVED** [review]

**Pull request closed without merging**